app/notify_client/service_api_client.py

```diff
@@ -53,6 +53,18 @@ def get_service_notification_statistics_by_day(
             "/service/{0}/statistics/{1}/{2}".format(service_id, start_date, days),
         )["data"]
 
+    def get_user_service_notification_statistics_by_day(
+        self, service_id, user_id, start_date=None, days=None
+    ):
+        if start_date is None:
+            start_date = datetime.now().strftime("%Y-%m-%d")
```

Comment:
This may be totally fine as is, but I figured I'd double check!  Should we use the `utc_now` utility method here instead?

Reply:
Great catch!!!